documentation/TechnicalDesignDocument.tex

```diff
@@ -96,6 +96,26 @@
     In this section build automation and its constituent components will be discussed.
     How we applied it, the software used and the targets and platforms supported are among a few.
 
+    %------------------------------------------------------------------------------------------------------------------------------------------------------- Continuing Development
+    \section{Continuing Development}
+    This section describes the plan on how to continue with development in the best way.
+    The team worked on quite some user stories from april 9th, to june 15th.
+    The following functionalities are most critical at the moment to start working on:
+    \myparagraph{Instantiating maramafications from a .mar file.}
+    A beginning is made at implementing the .mar file.
+    The .mar file is basically a json file with some agreements on structure.
```

Comment:
Isn't a .mar file a .obj file with a json header?

Reply:
Nope, from my understanding the .obj files are linked or embedded from inside the .mar file. Is this corrent @ThijsvanDam?